torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -549,9 +549,15 @@ def prepare_for_partitioner(mod, num_primals, num_fw_outputs):
                     joint_hop_gm, num_fw_inputs, num_fw_outputs
                 )
 
+                # TODO: invoke_subgraph should track which of its inputs static indices
+                # so it can propagate them to the partitioner (and use in cudagraphs)
+                static_input_indices: list[int] = []
```

Comment:
dumb qn: How does dynamo determine what are considered static inputs? Is it just nn.Parameters or are captured globals also considered to be static?

Reply:
I think that dynamo only does this marking for graph inputs of type `torch.nn.Parameter` today - so other global tensors used in some compiled code will get lifted as graph inputs, but not marked static unless the user explicitly does that marking

Also if you use the `torch.tensor()` constructor directly in a compiled region, we will end up burning it into the graph as a constant (not a graph input)